### PR TITLE
cd, echo, git, link, mkdir: add Serbian translation

### PR DESCRIPTION
**Pull request by nebocoder**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.sr/common/git.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# git
+
+> Distribuirani sistem kontrole verzija.
+> Neke podkomande kao što je `git commit` imaju svoj primer u dokumentaciji.
+> More information: <https://git-scm.com/>.
```
Comment (k-stz) [inline pages.sr/common/git.md:5]: translate as well to: "Više informacija na"

Comment (nebocoder) [reply pages.sr/common/git.md]: Thanks for noticing it, I fixed it in both places.

File: pages.sr/common/cd.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# cd
+
+> Menja trnutnu radnu površinu..
+> Više informacija na: <https://man.archlinux.org/man/cd.n>.
```
Comment (navarroaxel) [inline pages.sr/common/cd.md:4]:
```suggestion
> Više informacija na: <https://manned.org/cd>.
```

File: pages.sr/common/cd.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Ulazi u dati direktorijum:
+
+`cd {{path/to/directory}}`
```
Comment (navarroaxel) [inline pages.sr/common/cd.md:8]: Please translate the arguments for get this easier for non-English speakers. Could this `path/to/directory` be translated?

File: pages.sr/common/echo.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`echo "{{My path is $PATH}}"`
+
+- Prikazuje poruku bez dodatne linije:
+
+`echo -n "{{Hello World}}"`
+
+- Dodaje poruku u fajl:
+
+`echo "{{Hello World}}" >> {{file.txt}}`
```
Comment (navarroaxel) [inline pages.sr/common/echo.md:20]: could you translate the filename?

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages.sr/common/cd.md (hunk 3 above)
Comment (nebocoder) [reply pages.sr/common/cd.md:8]: I wasn't sure about touching arguments, if you say it's okay I think I could translate it in such a way that it keeps it's meaning.

File: pages.sr/common/echo.md (hunk 4 above)
Comment (nebocoder) [reply pages.sr/common/echo.md:20]: Do I touch "Helo World"?

Comment (navarroaxel) [reply pages.sr/common/echo.md:20]: yeah, why not?

File: pages.sr/common/cd.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# cd
+
+> Menja trnutnu radnu površinu..
```
Comment (navarroaxel) [inline pages.sr/common/cd.md:3]: A typo here
```suggestion
> Menja trnutnu radnu površinu.
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**